src/Phantoman.php

```diff
@@ -34,6 +34,11 @@ public function __construct($config, $options)
             $this->config['path'] = "vendor/bin/phantomjs";
         }
 
+        // Add .exe extension if running on the windows
+        if ($this->isWindows() && file_exists(realpath($this->config['path'] . '.exe'))) {
+            $this->config['path'] .= '.exe';
+        }
```

Comment:
This `.exe` appending needs to happen after the "is directory" check on line 43 in order to ensure that windows setups that use *just* the folder will get the correct .exe name.

### Steps that would happen on windows to support the old directory config

1. Check if the configured path is a directory
    - If so, append `/phantomjs` to the end of the path
2. Check if we're on windows
    - If so, append `.exe` to the end of the path.

From `vendor/bin`, we should end up with `vendor/bin/phantomjs.exe` on Windows.

Can you make that change and let me know if setting the path to `vendor/bin` on Windows works for you? I'd like to ensure the old methods work. You will get the deprecation warning but that's what we want in that case.

After you ensure that works, you can set it to the normal `vendor/bin/phantomjs` etc.

Thank you for you help!